plugins/couchbase/src/main/java/org.radargun.service/CouchbaseService.java

```diff
@@ -0,0 +1,45 @@
+package org.radargun.service;
+
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.CouchbaseCluster;
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.Lifecycle;
+
+@Service(doc = "Couchbase")
+public class CouchbaseService implements Lifecycle {
+
+   protected Cluster cluster;
+   protected Bucket bucket;
+
+   @Property(doc = "Bootstrap nodes")
+   protected String nodes = "localhost";
+
+   @Property(doc = "Username")
+   protected String username = "Administrator";
+
+   @Property(doc = "Password")
+   protected String password = "password";
+
+   @Property(doc = "Bucket name")
+   protected String bucketName = "default";
+
+   @Override
+   public void start() {
+      cluster = CouchbaseCluster.create(nodes);
```

Comment:
I think since this is just doing sync kv ops, we can pass in the environment and set the following:

.callbacksOnIoPool(true)
.runtimeMetricsCollectorConfig(DefaultMetricsCollectorConfig.disabled())
.networkLatencyMetricsCollectorConfig(DefaultLatencyMetricsCollectorConfig.disabled())